axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveTemplateService.java

```diff
@@ -416,6 +421,22 @@ public List<Long> generateMove(LocalDate moveDate, List<HashMap<String, Object>>
     return moveList;
   }
 
+  protected Account getAccountTypeAccount(
+      List<MoveTemplateLine> moveTemplateLines, String accountType) {
+    Account account = null;
+
+    Optional<MoveTemplateLine> moveTemplateLine =
+        moveTemplateLines.stream()
+            .filter(
+                ml -> ml.getAccount().getAccountType().getTechnicalTypeSelect().equals(accountType))
+            .findFirst();
+    if (moveTemplateLine.isPresent()) {
+      account = moveTemplateLine.get().getAccount();
+    }
+
+    return account;
```

Comment:
I think this can be replaced by directly returning the result of the stream ending with `.map(MoveTemplateLine::getAccount).findFirst().orElse(null)`
Also `accountType` is not a required field